modules/claims_api/config/schemas/v2/request_bodies/disability_compensation/request.json

```diff
@@ -3,1032 +3,1011 @@
   "description": "Claims API 526 Schema",
   "type": "object",
   "additionalProperties": false,
-  "required": ["data"],
+  "required": [
+    "veteranIdentification",
+    "serviceInformation",
+    "disabilities",
+    "claimantCertification",
+    "claimProcessType"
+  ],
   "properties": {
-    "data": {
+    "claimDate": {
```

Comment:
Are the docs changing to show something else (just not sure what is coming in any subsequent PRs) or is there a new format for the request body being sent in?  (coudn't find that in the ticket but just let me know)

<img width="1319" alt="Screenshot 2023-08-14 at 4 21 02 PM" src="https://github.com/department-of-veterans-affairs/vets-api/assets/129893414/67e72bb0-9dce-4ce5-9fc7-07f4845550e7">